src/ivc/incrementally_verifiable_computation.rs

```diff
@@ -72,44 +72,44 @@ where
     C2::Base: PrimeFieldBits + FromUniformBytes<64>,
 {
     pub fn new<const T: usize, RP1, RP2>(
-        _pp: &PublicParams<C1, C2, RP1, RP2>,
+        _pp: &PublicParams<'_, A1, A2, T, C1, C2, SC1, SC2, RP1, RP2>,
```

Comment:
I didn't understand the question, we have two ARITYs, two Step Circuit and two of each, so that's why there are two generics each

Reply:
Since previously the generic is C1, C2, RP1, RP2, I just wonder which changes require we add more generics here. Trying to have a better understanding of the change

Reply:
Well we now have to pass inside the step circuit to configure the table data, and also store inside the step-config, because now we will no longer call the configuration, but only clone the finished td

That's why we had to put all the generics we have in here.